tests/run-tests.sh

```diff
@@ -3,45 +3,55 @@
 # Run a bunch of shell-based tests in isolated environments using Docker.
 #
 set -e
-REPOSITORY=/go/src/github.com/dcoker/biscuit/
-if [ "${CONTINUOUS_INTEGRATION}" != "true" ]; then
-  echo __ Running with credentials from biscuit-testing profile.
-  AWS_ACCESS_KEY_ID="$(aws configure --profile biscuit-testing get aws_access_key_id)"
-  AWS_SECRET_ACCESS_KEY="$(aws configure --profile biscuit-testing get aws_secret_access_key)"
-fi
-# Test keys created with `biscuit kms init`.
-# user/cli-integration-test granted usage of keys via `biscuit kms edit-key-policy`.
-AWS_ACCOUNT=872957446280
-AWS_REGION=us-west-1
-KEY1=8be1cc5b-3c70-4295-acad-e497dd421961
-ARN1=arn:aws:kms:us-west-1:${AWS_ACCOUNT}:key/${KEY1}
-ARN1_REGION=us-west-1
-KEY2=37cca9a8-d2d0-43b8-9363-bb8320176cee
-ARN2=arn:aws:kms:us-west-1:${AWS_ACCOUNT}:key/${KEY2}
-ARN2_REGION=us-west-1
+
+function aws() {
+  docker run --network localstack -e AWS_ACCESS_KEY_ID=test -e AWS_SECRET_ACCESS_KEY=test --rm amazon/aws-cli  --endpoint=http://localstack:4566 "$@"
```

Comment:
this assumes a `localstack` network, it can be spun up with `make localstack` that will set up what is needed to run these tests.